code/game/jobs/job/offcolony.dm

```diff
@@ -119,6 +119,46 @@
 	icon_state = "player-black"
 	join_tag = /datum/job/off_colony_herbalist
 
+/datum/job/off_colony_technician
```

Comment:
A whole new job for this seems...unnecessary. It also goes against their general 'thing' as outdoorsmen

Reply:
I dunno. Mitten said to create one instead of giving hunters a more general skillset

Reply:
I suppose theses are two equal solutions to a problem that I'm...not sure Hunters have.